src/Components/Input.tsx

```diff
@@ -146,6 +148,13 @@ export class Input extends React.Component<InputProps, InputState> {
               ? this.getRightViewForPassword()
               : !!rightView && rightView}
           </InputContainer>
+          {!error && showPasswordMessage ? (
```

Comment:
@kanaabe Why not just use the standard `error` field/system? Did it not work?

Reply:
Ah, because this is separate from an error - we want to show that password message always, _unless_ there is an error. Is that what you're wondering or something else?